mir_eval/util.py

```diff
@@ -715,7 +720,7 @@ def filter_kwargs(function, *args, **kwargs):
     function_args = inspect.getargspec(function).args
     # Construct a dict of those kwargs which appear in the function
     filtered_kwargs = {}
-    for kwarg, value in kwargs.items():
+    for kwarg, value in list(kwargs.items()):
```

Comment:
Either way is fine, as long as it works.  Adding six as a dependency is not a big deal as it's just a single Python file.